src/imitation/scripts/config/multi_expert_demos.py

```diff
@@ -0,0 +1,32 @@
+import math
+import multiprocessing
+import os.path as osp
+
+import sacred
+
+from imitation.util import util
+
+multi_expert_demos_ex = sacred.Experiment("multi_expert_demos")
+
+
+@multi_expert_demos_ex.config
+def config():
+  csv_config_path = "FILL ME"  # Config file
+  n_workers = math.ceil(
```

Comment:
Why is the integer ceiling more appropriate than the floor here? (Note for floor you can just write `multiprocessing.cpu_count() // 4`.)

Reply:
:+1: makes sense